shapeshift/web
# feat: programmatic FoxPage descriptions

## Description

Following the latest spec, this uses programmatic descriptions depending on the currently selected tab:

- Uses the custom FOX description provided in spec for FOX
- Uses FOXy asset description for FOXy (PRd in asset-service: https://github.com/shapeshift/lib/pull/710)
- Removes description trim as per latest spec

## Notice




## Pull Request Type


## Issue (if applicable)

N/A

## Risk

N/A

## Testing

- Go to FOX tab, the description should be `Since 2019, our shapeshifting FOX Token has been offering users an ever-expanding world of utility and advantages. Today, our ERC-20 governance token not only enables you to influence the future of ShapeShift through your vote, you also have an ever-expanding universe of investing opportunities. Invest, track, and manage your FOX holdings here.`
- Go to FOXy tab, the description should be `4. Below that will be a description COPY: FOXy is a rebasing ERC-20 Token, which you can exchange 1:1 with FOX Tokens, that offer both yield and profit redistribution from the ShapeShift DAO. Get more out of your FOX without losing your governance power.`

## Screenshots (if applicable)
